assets/schema_input.json

```
@@ -0,0 +1,58 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/nf-core/metapep/master/assets/schema_input.json",
+    "title": "nf-core/metapep pipeline - params.input schema",
+    "description": "Schema for the file provided with params.input",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "condition": {
+                "type": "string",
+                "pattern": "^\\S+$",
+                "errorMessage": "Condition name must be provided and cannot contain spaces"
+            },
+            "type": {
+                "type": "string",
+                "pattern": "^(bins)|(assembly)|(taxa)$",
```

```suggestion
                "enum": ["bins", "assembly", "taxa"],
```
There's a built-in method called `enum` which does exactly this and provides some better error messages than `pattern` :)